code/modules/mob/living/silicon/robot/drone/drone.dm

```diff
@@ -321,10 +324,20 @@
 /mob/living/silicon/robot/drone/Bumped(atom/movable/AM)
 	return
 
+var/list/pullable_drone_items = list(
```

Comment:
Good but you should probably declare this on drones, as right now its just hanging around. I'm not sure how this would behave actually. 

What I mean is something like this:

```js
/mob/living/silicon/robot/drone
	name = "drone"
    ... // other stuff

	var/emagged_time
    var/list/pullable_drone_items = list(
        ...
    )

Reply:
Seems to work just fine ingame as it is. *shrug

It should be like this then?
```
        var/last_reboot
        var/emagged_time
	var/list/pullable_drone_items = list(
	/obj/item/pipe,
	/obj/structure/disposalconstruct,
	/obj/item/stack/cable_coil,
	/obj/item/stack/rods,
	/obj/item/stack/sheet,
	/obj/item/stack/tile
	)

Reply:
Yeah, but indent all the entries in the list to the right one tab. Also my indenting got messed up in my last comment, fixed it though.